caching-big-repositories_5bf03fb92c7d3a31944e139d.md

```diff
@@ -0,0 +1,167 @@
+
+* [Overview](#overview)
+* [The scenario](#the-scenario)
+* [The logic behind the recipe](#the-logic-behind-the-recipe)
+* [An example project](#an-example-project)
+* [See Also](#see-also)
+
+## Overview
+
+The purpose of this document is to provide a recipe for caching big
+repositories in order to save time.
+
+## The scenario
```

Comment:
This section and the overview should be one section without a subheader. Just jump straight to the purpose of the guide.